src/docs/product/security/mobile-privacy.mdx

```diff
@@ -54,3 +54,12 @@ No. Sentry does not cause your app to contain any ads.
 ### What do I need to disclose to Google Play?
 
 You would need to disclose and/or include in your app privacy policy all types of data you are collecting through your app, including data you are sending to Sentry.
+
+## Sentry SDKs for third-party SDKs and libraries
+
+Sentry SDKs are not officially supported and recommended for use in third-party SDKs and libraries due to technical limitations.
+
+- Sentry SDKs are not designed to be used in third-party SDKs and libraries.
+- Version conflicts may occur if multiple Sentry SDKs are installed in the same application.
+- [Signal handlers](https://en.wikipedia.org/wiki/C_signal_handling) may overwrite each other leading to one of the Sentry SDKs not operating normally if multiple Sentry SDKs are installed in the same application.
+- Third-party SDKs and libraries may leak users' PII data to the third-party's Sentry organization.
```

Comment:
Since this bullet is the only one related to the topic of this page, I think it could be moved to the general section, perhaps included as a warning under the first question.

Reply:
Moved to the suggested page.